core/src/main/scala/com/evolutiongaming/catshelper/FromFuture.scala

```diff
@@ -54,20 +53,41 @@ object FromFuture {
     }
   }
 
+  def asyncFromFuture[F[_]: Async]: FromFuture[F] = {
```

Comment:
1. I suggest to rename to `fromAsync`
2. removing `implicit` in the `lift` method is not a breaking change, so you still can make new `fromAsync` method an implicit :)

Reply:
Making the method implicit in the same scope as old `lift` will make them clash. Might be I should introduce new object like `Implicits` with new implicit method, but definitely not bring it on the same scope as old `lift`

Reply:
but you should remove `implicit` keyword from old `lift`!

Reply:
Ah, got it. I was thinking that removing `implicit` from old `lift` is braking change that you'd like to avoid.